### PR TITLE
Fixed ether command on Ethereum Client APIs

### PR DESCRIPTION
**Pull request by pglekshmi**
Fixed ether command on Ethereum Client APIs

## Description

Changed
    ethers.providers.Web3Provider(window.ethereum) --> ethers.BrowserProvider(window.ethereum)  & 
    Wallet.fromMnemonic(mnemonics) --> Wallet.fromPhrase(mnemonics)

### CODE REVIEW

File: public/content/developers/docs/apis/javascript/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Using providers, these libraries allow you to connect to Ethereum and read its d
 ```js
 // A Web3Provider wraps a standard Web3 provider, which is
```
Comment (wackerow) [inline public/content/developers/docs/apis/javascript/index.md:32]: Thanks @pglekshmi! Seeing support for this in the [v6 ethers docs](https://docs.ethers.org/v6/getting-started/)... Let's also update the comment so it aligns:

```suggestion
// A BrowserProvider wraps a standard Web3 provider, which is
```

File: public/content/developers/docs/apis/javascript/index.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Here's an examples from Ethers
 // Create a wallet instance from a mnemonic...
 mnemonic =
   "announce room limb pattern dry unit scale effort smooth jazz weasel alcohol"
-walletMnemonic = Wallet.fromMnemonic(mnemonic)
+walletMnemonic = Wallet.fromPhrase(mnemonic)
```
Comment (wackerow) [inline public/content/developers/docs/apis/javascript/index.md:83]: Looks good! ([supporting docs here](https://docs.ethers.org/v6/api/wallet/#Mnemonic_fromPhrase) for reference)

**Review by wackerow — APPROVED** [review]
Thanks @pglekshmi! Going to commit the adjustment to the comment then pull this in.

**Comment by wackerow** [comment]
@all-contributors please add @pglekshmi for maintenance

**Comment by pglekshmi** [comment]
Thank you for accepting pull request & looking forward to do more contributions

**Pull request merged**